test/integration/egress.go

```diff
@@ -65,4 +75,13 @@ func (t *egress) run() error {
 }
 
 func (t *egress) teardown() {
+	if !t.Egress {
+		return
+	}
+	if err := client.CoreV1().Services(t.Namespace).Delete("httpbin", &meta_v1.DeleteOptions{}); err != nil {
```

Comment:
(optional): you could use "kubectl delete" to avoid hardcoding the individual service names, e.g. 

```golang
if err := util.Run(mt.Sprintf("kubectl -n %s delete -f %s", externalServiceFile, t.Namespace)); err != nil {
    glog.Warning(err)
}
```